### PR TITLE
Alphabetical order of EVM chains and subchapters, summary bugs, valid…

### PR DESCRIPTION
**Pull request by Hactive808**
… hashes for klaytn and ETH Classic and fixes of few leftovers

### CODE REVIEW

**Comment by Hactive808** [comment]
Needs to be merged because of hotfix

**Review by Hactive808 — COMMENTED** [review]
hotfix needed

File: docs/rpc/evm-blockchains/arbitrum-rpc-documentation/network-state/eth_call.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/arbitrum-rpc-documentation/network-state/eth_call.md]: No ethereum here

File: docs/rpc/evm-blockchains/avalanche-rpc-documentation/network-state/eth_call.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/avalanche-rpc-documentation/network-state/eth_call.md]: no ethereum here

File: docs/rpc/evm-blockchains/ethereum-classic-rpc-documentation/network-state/eth_call.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/ethereum-classic-rpc-documentation/network-state/eth_call.md]: ethereum classic

File: docs/rpc/evm-blockchains/flare-rpc-documentation/network-state/eth_call.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/flare-rpc-documentation/network-state/eth_call.md]: no ethereum here

File: docs/rpc/evm-blockchains/haqq-rpc-documentation/network-state/eth_call.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/haqq-rpc-documentation/network-state/eth_call.md]: no ethereum

File: docs/rpc/evm-blockchains/horizen-eon-rpc-documentation/network-state/eth_call.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/horizen-eon-rpc-documentation/network-state/eth_call.md]: no ethereum

File: docs/rpc/evm-blockchains/klaytn-rpc-documentation/network-state/klay_call.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/klaytn-rpc-documentation/network-state/klay_call.md]: no ethereum

File: docs/rpc/evm-blockchains/optimism-rpc-documentation/network-state/eth_call.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/optimism-rpc-documentation/network-state/eth_call.md]: no ethereum

File: docs/rpc/evm-blockchains/polygon-rpc-documentation/network-state/eth_call.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+### JSON Examples
+
+#### Request
+
+```json
+{
+    "jsonrpc":"2.0",
+    "method":"eth_call",
+        "params":[{
+            "to": "0xD31a59c85aE9D8edEFeC411D448f90841571b89c", // Replace with the ERC-20 token contract address in this case wrapped SOL on Ethereum
```
Comment (Hathoriel) [inline docs/rpc/evm-blockchains/polygon-rpc-documentation/network-state/eth_call.md]: no ethereum

**Review by Hathoriel — APPROVED** [review]

**Pull request merged**